### DISCUSSION TITLE
Fetch all friendships at once

### DISCUSSION THREAD

**Discussion by murataksu740** [sbertix/Swiftagram#242] [Ideas]
https://i.instagram.com/api/v1/friendships/{id}/followers/?count=10000

?count=10000

This is a way to bring in the most friends at once. 10,000 is a limit based on my experience. Please don't forget to test it. good work.

**Comment by sbertix** [comment]
Hey @murataksu740, 

I'm already planning, for `6.*`, for the user to be able to inject whatever parameters to the underlying requests, even when said parameters exist but are undocumented and "unsafe" (as Instagram never directly relies on them in the client and might result in the account being flagged), like in this case. 

Thanks 💪
